src/main/java/com/example/TicketBookingApp/exception/BookingTimeException.java

```diff
@@ -0,0 +1,7 @@
+package com.example.TicketBookingApp.exception;
+
+public class BookingTimeException extends RuntimeException{
+    public BookingTimeException() {
+        super("Seats can be booked at latest 15 minutes before the screening begins");
```

Comment:
a jak ktoś zmieni te 15 minut to w ilu miejscach trzeba zmienić?

Reply:
Powinienem zdefiniować zmienną _MAX_TIME_BEFORE_START_SCREENING_. Zrobiłem to tylko w _ReservationService_, co też nie jest dobrym rozwiązaniem. Zmienna powinna być zdefiniowana w pliku .yml